index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/src/app/favicon.svg" />
```

Comment:
Make sure to remove reference to `favicon.svg` in your `index.html`, otherwise, you will get an error